config/default.js

```diff
@@ -93,4 +93,12 @@ module.exports = {
     max: '1500000',
     defaultValue: '150000',
   },
+
+  // number of onchain confirmations for the specified periods
+  // potentially needs multiple chain support (LTC)
+  lndTargetConfirmations: {
```

Comment:
I think this would be cleaner defined like:

```
fees: {
  fast: 1,
  medium: 3,
  slow: 6
}
```

So accessed like `config.fee.fast` etc.

This way, the three values are keyed on generic names with more flexible meaning, rather than being tied to a specific time frame like `halfHourConfCount `. Would make it possible to change the targets in the future without ending up with a mismatch between the far name and the value used.